articles/multifactor-authentication/ro.md

```diff
@@ -0,0 +1,225 @@
+---
+title: MFA on a Resource Owner endpoint
+description: How to enable MFA on the Resource Owner endpoint.
+---
+
+# MFA on a Resource Owner endpoint
+
+To enable MFA on the Resource Owner [/ro](/api/authentication#!#post--oauth-ro) endpoint, include the `options.mfa.active` flag in the connection and set it to `true`. 
```

Comment:
@mcastany Where is this flag? Its not in the endpoint call, is it a setting in the dashboard?